src/server.ts

```diff
@@ -181,10 +184,24 @@ const fetchVulnerabilities = async (reqData) => {
     const headers = {
         'Content-Type': 'application/json',
         'Authorization': 'Bearer ' + config.api_token,
+        'request_id': uuid.v4(),
     };
+
+    let manifestHash = crypto.createHash("sha1").update(reqData.filepath).digest("hex");

```

Comment:
manifestHash should be generated only once from sendDiagnostics, not on each batch.

Reply:
done.